assets/src/design-system/components/menu/list/components.js

```diff
@@ -43,36 +43,19 @@ export const ListItemLabel = styled.li`
 `;
 
 export const ListItem = styled.li(
-  ({ disabled, isSelected, theme }) => css`
+  ({ disabled, theme }) => css`
     display: grid;
     grid-template-columns: 16px 1fr;
     padding: 6px 2px 6px 8px;
     margin: 4px 8px;
     border-radius: ${theme.borders.radius.small};
     align-items: center;
-    background-color: ${isSelected
-      ? theme.colors.interactiveBg.brandNormal
-      : 'inherit'};
     cursor: ${disabled ? 'default' : 'pointer'};
 
     ${themeHelpers.focusableOutlineCSS(theme.colors.border.focus)};
 
-    &:focus {
-      background-color: ${isSelected
-        ? theme.colors.interactiveBg.brandNormal
-        : theme.colors.interactiveBg.brandHover};
-    }
-
     &:hover {
-      background-color: ${isSelected
-        ? theme.colors.interactiveBg.brandNormal
-        : theme.colors.interactiveBg.brandHover};
-    }
-
-    &:active {
-      background-color: ${isSelected
-        ? theme.colors.interactiveBg.brandActive
-        : theme.colors.interactiveBg.brandPress};
+      background-color: ${theme.colors.bg.tertiary};
```

Comment:
should we also have an active state here? I feel like most other button-like components have active states.

The context menu uses `secondaryHover` and `secondaryPress` for this 🤷

Reply:
I think this is a good call out, the styles on this just recently were updated - let's give design a chance to play with it and decide what they want moving forward as it's not documented in figma? This will be easier to probably demonstrate and get a grasp on that way.